ccx_keys/locator.py

```diff
@@ -1,6 +1,109 @@
 # -*- coding: utf-8 -*-
+import re
+
+from opaque_keys import InvalidKeyError
 from opaque_keys.edx.locator import CourseLocator
 
+from ccx_keys.key import CCXKey
+
+
+class CCXLocator(CourseLocator, CCXKey):
+    """Concrete implementation of an Opaque Key for CCX courses"""
+
+    CANONICAL_NAMESPACE = 'ccx-v1'
+    KEY_FIELDS = ('org', 'course', 'run', 'branch', 'version_guid', 'ccx')
```

Comment:
They match, but you could in fact just write this as `CourseLocator.KEY_FIELDS + ('ccx',)` to reduce the duplication.